src/app/beer_garden/scheduler.py

```diff
@@ -530,6 +540,69 @@ def create_job(job: Job) -> Job:
     return job
 
 
+def create_jobs(jobs: List[Job]) -> List[str]:
+    """Create multiple new Jobs and add them to the scheduler.
+
+    Args:
+        jobs: A list containing the `Job`s to be added
+
+    Returns:
+        A list containing the `id`s of the `Job`s that were added
+    """
+    job_collection_name = "job"
+    id_key, oid_key = "_id", "$oid"
+
+    # we want the addition of Jobs to obey ACID: either all will succeed or none
+    # will succeed; hence we skip using mongoengine and use the pymongo library
+    # directly
+    the_connection: pymongo.MongoClient = mongoengine.get_connection()
+    the_db: pymongo.database.Database = mongoengine.get_db()
+
+    session: ClientSession
+    with the_connection.start_session() as session:
+        job_collection: Collection = the_db.get_collection(job_collection_name)
+
+        to_mongo_caller = methodcaller("to_mongo")
+        job_bson_objects = map(to_mongo_caller, map(from_brewtils, jobs))
+
+        # this does not work for mongodb running standalone
```

Comment:
https://docs.mongodb.com/manual/core/read-isolation-consistency-recency/#std-label-sessions

I am not a database expert so I cannot remark on whether this fulfills all the requirements to be properly called a *transaction* from a database-theoretic point of view. I think it probably isn't but I also think the distinction is mostly immaterial for what we're trying to do.

Wrapping the code in a session and using `insert_many` instead of `insert_one` ensures everything in the provided list is added to the db or nothing in the list is added. I.e., atomicity of the operation over a list. I think this suffices for our current uses, so I'd like to keep it.

I've also added some more comments to explain why commented-out code is being retained, a habit I think is bad in most cases (but not this one, obviously).